nflow-tests/src/test/java/io/nflow/tests/AbstractNflowTest.java

```diff
@@ -118,7 +120,16 @@ protected ListWorkflowInstanceResponse getWorkflowInstance(int id, String expect
     return wf;
   }
 
-
+  protected ListWorkflowInstanceResponse getWorkflowInstanceWithTimeout(int id, String expectedState, Duration timeout) throws InterruptedException {
```

Comment:
Duration timeout -> int timeoutSeconds? now there is ofSeconds(x) everywhere in tests

Reply:
The `Duration` can be of any unit. This code does now know the unit is was based on.